lib/generators/shopify_app/rotate_shopify_token_job/templates/rotate_shopify_token.rake

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+namespace :shopify do
+  desc "Rotate shopify tokens for all active shops"
+  task :rotate_shopify_tokens, [:refresh_token] => :environment do |_t, args|
+    all_active_shops.each do |shop|
+      Shopify::RotateShopifyTokenJob.new.perform(
+        shop_domain: shop.shopify_domain,
```

Comment:
I think this makes the assumption that apps are calling the primary domain `shopify_domain` locally. Some apps might be using `shop_domain` or `domain` or `primary_domain`.

Reply:
`shopify_domain` is the column that the Shop model generator in `shopify_app` provides by default.

The rake task is only meant to be an example implementation, and isn't intended to be able to be used out of the box by all apps. The job is what we are really aiming to make re-usable here.